README.md

```diff
@@ -65,6 +65,25 @@ a one-time manual copy of this dataset to your project location using the [BigQu
 1. Run `bash transfer_dataset.sh <project_id> <dataset> <location>`
 </p></details>
 
+### Hands-on Tutorials
+
+**TODO:** Update cloudshell_git_repo from teamdatatonic to GoogleCloudPlatform X 2 + cloudshell_git_branch to main X 2
+
+Click "Open in Cloud Shell" for a tutorial on how to get started with Vertex Pipeline Templates for
+1. Building a **Training pipeline**
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/teamdatatonic/vertex-pipelines-end-to-end-samples&cloudshell_git_branch=tutorial/cloud-shell-train-predict-pl&cloudshell_workspace=.&open_in_editor=README.md&tutorial=docs/TUTORIAL_TRAINING.md)
```

Comment:
are we planning to keep the branch `tutorial/cloud-shell-train-predict-pl` open or is there a better way to pin the right version of the repo?

Reply:
Git branch needs to be explicitly mentioned in this hyperlink to the tutorial.
To that effect, this will need to be changed to `main`/`master` before merging in with the Official Google repo. The same is captured in the TODO above

Reply:
when you link to the main branch and any code is updated through a feature branch on main while the tutorial remains stale, this tutorial will reference the updated code and fail in the cloud console. @browningjp-datatonic any advice?

Reply:
@felix-datatonic Update `pull_request_template.md` to ensure that a developer updates the tutorials (if applicable). hope this suffices

Reply:
Given the immediate restructuring work, I would not like to make this part of the PR checklist. It would add an overhead to maintain the tutorial in every PR. Rather I prefer to pin the tutorial to the current version of the templates and use this in the tutorial. Then we can update the tutorial at selected occasions (e.g. major releases).

Reply:
@browningjp-datatonic thinking about a repo link in cloud shell similar to https://github.com/teamdatatonic/vertex-pipelines-end-to-end-samples/tree/8fe053d68f6e486044e3c618fbafe3fc0cffd81d

Reply:
Think I agree with Rahul on this one. I think pinning to a specific repository version will get very confusing. How much of a burden will it actually be to keep the tutorial up-to-date?

The other option we could consider - remove the cloud shell tutorials from this repo, and instead host them in a separate Datatonic repo? Then we can link to whatever version of the main repo we like

Reply:
Keeping the tutorial up to date as part of each PR will vary in complexity - especially if the PR changes are significant or the contributor does not understand (yet) how to maintain the tutorials. Tutorials can be treated similarly to documentation which always refer to a specific version of a project. I was hoping on a suggestion e.g. by using release tags @browningjp-datatonic as my comment above is a very crude method (which I wouldn't like to implement).

Reply:
@felix-datatonic Pinned the tutorial to release tag v1. Releases in this repo will be created from `dev`.
Only catch is that once this PR is merged into `dev`, release v1 needs to be deleted & recreated so that it contains the necessary files for the tutorial